README.md

```diff
@@ -40,11 +43,17 @@ https://github.com/frej/fast-export.git).
   branch from the resulting Git repository by running `git branch -D subfeature`.
 
 ## Warnings
-- Both scripts generate a lot of output on stdout and stderr. If you want
-  to analyze it, better redirect it into files.
+- The `run-cleanup.sh` and `run-conversion.sh` scripts generate a lot of output 
+  on stdout and stderr. If you want to analyze it, better redirect it into files.
```

Comment:
The readme should have a section "Troubleshooting" that mentions what to do if things go wrong. As a general suggestion, I would say that if the script doesn't work, it is good to run the steps individually and carefully inspect the intermediate results. If this doesn't help, I'd suggest pulling in (and possibly merging) the latest changes from hg.fast-downward.org before running the scripts. That should help with some issues where the stripping would fail (for example, if a repository has only one half of the branch that we want to splice together).

Reply:
I added a small section